src/main/java/com/example/petbuddybackend/service/user/CaretakerService.java

```diff
@@ -38,7 +38,7 @@ public class CaretakerService {
     private final ClientService clientService;
     private final UserService userService;
 
-    @Transactional
+    @Transactional(readOnly = true)
```

Comment:
o fajnie, pierwszy raz widze, ze ktos uzyl opcji w transactionalu

Reply:
Akurat czytałem o tym to uznałem że dodam